templates/orbital-sync.xml

```diff
@@ -0,0 +1,63 @@
+    <Config Name="secondary pihole 5 password" Target="SECONDARY_HOST_5_PASSWORD" Default="" Mode="" Description="The password used to log in to the admin interface." Type="Variable" Display="advanced" Required="false" Mask="true" />
+    <Config Name="secondary pihole 6 baseurl" Target="SECONDARY_HOST_6_BASE_URL" Default="" Mode="" Description="The base URL of your secondary Pi-hole, including the scheme (HTTP or HTTPS) and port but not including a following slash. i.e. 'http://192.168.1.8' or 'https://pihole7.example.com'" Type="Variable" Display="advanced" Required="false" Mask="false" />
+    <Config Name="secondary pihole 6 password" Target="SECONDARY_HOST_6_PASSWORD" Default="" Mode="" Description="The password used to log in to the admin interface." Type="Variable" Display="advanced" Required="false" Mask="true" />
+    <Config Name="secondary pihole 7 baseurl" Target="SECONDARY_HOST_7_BASE_URL" Default="" Mode="" Description="The base URL of your secondary Pi-hole, including the scheme (HTTP or HTTPS) and port but not including a following slash. i.e. 'http://192.168.1.9' or 'https://pihole8.example.com'" Type="Variable" Display="advanced" Required="false" Mask="false" />
+    <Config Name="secondary pihole 7 password" Target="SECONDARY_HOST_7_PASSWORD" Default="" Mode="" Description="The password used to log in to the admin interface." Type="Variable" Display="advanced" Required="false" Mask="true" />
+    <Config Name="secondary pihole 8 baseurl" Target="SECONDARY_HOST_8_BASE_URL" Default="" Mode="" Description="The base URL of your secondary Pi-hole, including the scheme (HTTP or HTTPS) and port but not including a following slash. i.e. 'http://192.168.1.10' or 'https://pihole9.example.com'" Type="Variable" Display="advanced" Required="false" Mask="false" />
+    <Config Name="secondary pihole 8 password" Target="SECONDARY_HOST_8_PASSWORD" Default="" Mode="" Description="The password used to log in to the admin interface." Type="Variable" Display="advanced" Required="false" Mask="true" />
+    <Config Name="secondary pihole 9 baseurl" Target="SECONDARY_HOST_9_BASE_URL" Default="" Mode="" Description="The base URL of your secondary Pi-hole, including the scheme (HTTP or HTTPS) and port but not including a following slash. i.e. 'http://192.168.1.11' or 'https://pihole10.example.com'" Type="Variable" Display="advanced" Required="false" Mask="false" />
+    <Config Name="secondary pihole 9 password" Target="SECONDARY_HOST_9_PASSWORD" Default="" Mode="" Description="The password used to log in to the admin interface." Type="Variable" Display="advanced" Required="false" Mask="true" />
+    <Config Name="interval minutes" Target="INTERVAL_MINUTES" Default="30" Mode="" Description="How long to wait between synchronizations. Defaults to five minutes. Remember that the DNS server on your secondary servers restarts everytime a sync is performed. i.e. '5', '30', or '1440'" Type="Variable" Display="always" Required="false" Mask="false" />
```

Comment:
```suggestion
    <Config Name="interval minutes" Target="INTERVAL_MINUTES" Default="30" Mode="" Description="How long to wait between synchronizations. Defaults to five minutes. Remember that the DNS server on your secondary servers restarts everytime a sync is performed. i.e. '5', '30', or '1440'" Type="Variable" Display="always" Required="false" Mask="false">30</Config>
```

Any field with a `Default` should have that default auto-populated.

Reply:
Is this _in addition_ to `Default="30"` as in your suggestion or is it supposed to in place of:
```suggestion
    <Config Name="interval minutes" Target="INTERVAL_MINUTES" Mode="" Description="How long to wait between synchronizations. Defaults to five minutes. Remember that the DNS server on your secondary servers restarts everytime a sync is performed. i.e. '5', '30', or '1440'" Type="Variable" Display="always" Required="false" Mask="false">30</Config>
```

Reply:
I pushed up the former, but I'm happy to do the latter if necessary!

Reply:
Yes, in addition.